docs/products/kafka/howto/connect-with-java.md

```diff
@@ -99,17 +104,54 @@ properties.put("ssl.truststore.type", "JKS");
 properties.put("ssl.truststore.location", "{TRUSTSTORE_LOCATION}");
 properties.put("ssl.truststore.password", "{TRUSTSTORE_PASSWORD}");
 
-// create a producer with String Serializer for key and value
+// Create a producer with StringSerializer for key and value
 KafkaProducer<String, String> producer = new KafkaProducer<>(properties, new StringSerializer(), new StringSerializer());
 ```
 
+### With Schema Registry (Karapace)
+
+```java
+Properties properties = new Properties();
+properties.put("bootstrap.servers", "{HOST}:{SSL_PORT}");
```

Comment:
Regards all the code blocks in the document: normally in the Aiven docs, ALL_CAPS itself is used for placeholders with no need for curly braces {}. Consider removing them.
```suggestion
properties.put("bootstrap.servers", "HOST:SSL_PORT");
```